### PR TITLE
Fix image filename lookups for default emoji.

### PR DESCRIPTION
**Pull request by wearhere**

### CODE REVIEW

**Comment by mislav** [comment]
Thanks. To be clear, this normalizes the filenames of default, yellow emoji to not have ".0" suffix, right?

**Comment by wearhere** [comment]
Yup. And when I say "fixes" the lookups, that's cause based on _just_ the base emoji, you'd expect its name to not have any suffix—you wouldn't have any way of knowing that it even _could_ be modified. 

(Without modifying the emoji database to hardcode that. I tried rerunning the database generation script to see if maybe lookups would work afterward and they didn't seem to… not sure if I regenerated it right.)

**Comment by wearhere** [comment]
This PR does not yet support lookups for the _non_-default emoji. I've changed the name and commit message to be more clear.

File: lib/emoji/extractor.rb [hunk 1]
```diff
@@ -23,6 +23,12 @@ def extract!
         hexes = ttf_name.split('_').map { |n| n.gsub(/^u/, '').downcase }
 
         if emoji_char_codes.include?(hexes.first.to_i(16))
+          # When an emoji supports skin-tone modifiers, all variants--including
+          # the unmodified "base" emoji--are suffixed with a "." + a number:
+          # 0 for the base, then 1-5 for Fitzpatrick type
+          # (https://en.wikipedia.org/wiki/Fitzpatrick_scale) 2-6. We remove the
```
Comment (mislav) [inline lib/emoji/extractor.rb]: What does "2-6" mean here?

Comment (wearhere) [inline lib/emoji/extractor.rb]: `1f44b.1.png` corresponds to :wave: in shade Fitzpatrick type II, `1f44b.2.png` to Fitzpatrick type III, and so on.

Comment (mislav) [inline lib/emoji/extractor.rb]: Thanks for the clarification. Can you make this more explicit so future readers don't get confused like me? Maybe word it like:

> Then 1-5 which maps to Fitzpatrick types 2-6 (link)

Comment (wearhere) [inline lib/emoji/extractor.rb]: No problem. Updated.

**Comment by mislav** [comment]
Thanks!

**Pull request merged**